src/docs/reference-manual/syntax.Rmd

```diff
@@ -72,17 +72,17 @@ var_type ::= 'int' range_constraint
            | 'corr_matrix' '[' expression ']'
            | 'cov_matrix' '[' expression ']'
 
-constraint ::= range_constraint | '<' loc_scale '>')
+constraint ::= range_constraint | '<' offset_multiplier '>')
```

Comment:
the src/docs dir should have been deleted by now.
changes to the documentation needs to be made in the `docs` repo.
if it's a question of changing `loc_scale` to `offest_multiplier`, I could do that.

Reply:
done.